cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -82,9 +92,12 @@ object ReporterCommand : CommandWithHelp() {
 
         val ortResult = ortResultFile.readValue<OrtResult>()
 
+        val resolutionProvider =
+            DefaultResolutionProvider(ortResult.repository.config.resolutions ?: Resolutions(), resolutionsFile)
```

Comment:
Here it occurs to me that we should probably have a `ResolutionFileProvider` and a list of `ResolutionProviders` (with two instanced of `ResolutionFileProvider`) instead of passing two files to a `DefaultResolutionProvider`.

Reply:
Working on it.

Reply:
Took a different approach, see `DefaultResolutionProvider` in the updated code.